### ISSUE TITLE
Modal design: Need to scroll to see top/bottom of modal with important user actions

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/platform-forms-client#4728]
## Description
In long modals, the need to scroll obfuscates the heading/close button and the CTA buttons.

## Steps to reproduce
1. Go to a draft form
2. Select API delivery in Settings
3. Create an API key
4. See need to scroll in modal 
5. Test out filling out a response
6. Go to responses page
7. Select download to open another modal and see the scroll hide the CTA buttons

## Details
- **Browser**: Edge
- **Operating system**: Windows
- **Environment**: Staging
- **Language**: English
- **Viewport**: Tablet

## Expected behaviour
Consider freezing the frame with the heading/title/close button statically locked and the CTAs also always visible

## Screensh
![scroll](https://github.com/user-attachments/assets/7f11900f-160f-4b54-bf3f-f928def1bdb8)
![no scroll](https://github.com/user-attachments/assets/00c16e55-cced-489a-ac16-1e6bff965181)
ots or videos

**Comment by timarney** [comment]
@Abi-Nada 

Perhaps we look at this card for Dialogs in general vs the API ... we re-use the dialog across the app.

**Comment by Abi-Nada** [comment]
Makes sense! Moving it out of Sprint Backlog so we can prioritize API-specific work accordingly

**Comment by Abi-Nada** [comment]
Modal dialog scrolling issue combined in 5099 ^
